core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -66,15 +74,18 @@ THE SOFTWARE.
                 <th width="32" tooltip="${%Uncheck to disable the plugin}">${%Enabled}</th>
                 <th initialSortDir="down">${%Name}</th>
                 <th width="1">${%Version}</th>
-                <th width="1">${%Previously installed version}</th>
-                <th width="1">${%Uninstall}</th>
+                <l:isAdmin>
+                  <th width="1">${%Previously installed version}</th>
```

Comment:
Why not allowing the previous version? Do you think it's a risk?

Reply:
it's just a button that says 'Downgrade to previous version', not useful to read only users

Reply:
The only UI element shown there is the "Downgrade" button, and the permission doesn't allow taking action, so there wouldn't be a UI.

An argument can be made to show it in text, but the question is how useful that would be – it's not really part of configuration, after all.

Reply:
I had in my mind this was only text. Fair enough!